services/src/services/errorTelemetry/errorTelemetry.hooks.js

```diff
@@ -8,7 +19,20 @@ class ErrorTelemetryHooks {
         all: [],
         find: [],
         get: [],
-        create: [],
+        create: [
```

Comment:
The reason I refactored many of the hooks which I wrote into this class structure with `getHooks()` was so  that they could be properly unit tested.

Will you refactor this anonymous function into a class function and add a couple tests for it?